playground.html

```diff
@@ -10,17 +10,21 @@
     <style>
         body {
             display: flex;
```

Comment:
I think it's not a good practice to use flex on the body.
Correct me if I'm wrong.